src/shared/constants.h

```diff
@@ -63,6 +63,9 @@ const double DRIBBLER_RPM_TO_RADIO_CONVERSION_FACTOR = 1.0 / 300.0;
 
 const double POSSESSION_TIMESTAMP_TOLERANCE_IN_MILLISECONDS = 10;
 
+// The maximum number of elements in the circular buffer for the motion controller
+const int CIRCULAR_BUFFER_MAX_NUM_ELEMENTS = 10;
```

Comment:
ditto comment above

Reply:
Apologies, what does this mean? That this comment is a duplicate from somewhere?

Reply:
nope my bad the comment order got messed up, was just referring to removing this, this is no longer relevant